tests/ui/test_capacity_breakdown_ui.py

```diff
@@ -0,0 +1,104 @@
+import pytest
+import logging
+
+from ocs_ci.ocs import constants
+from ocs_ci.framework.testlib import (
+    ManageTest,
+    tier2,
+    skipif_ui_not_support,
+    bugzilla,
+)
+from ocs_ci.helpers import helpers
+from ocs_ci.ocs.ui.validation_ui import ValidationUI
+
+
+logger = logging.getLogger(__name__)
+
+
+class TestCapacityBreakdownUI(ManageTest):
+    """
+    Test Capacity Breakdown UI
+
+    Test Process:
+    1.Create new project
+    2.Create pvc with relevant sc [ceph-rbd,cephfs]
+    3.Create pod
+    4.Run FIO
+    5.Verify new project created on Capacity Breakdown UI
+    6.Verify new pod created on Capacity Breakdown UI
+
+    """
+
+    @pytest.mark.parametrize(
+        argnames=["project_name", "pod_name", "sc_type"],
+        argvalues=[
+            pytest.param(
+                *["rbd1", "rbd2", constants.CEPHBLOCKPOOL_SC],
+                marks=pytest.mark.polarion_id("OCS-XXX"),
+            ),
+            pytest.param(
+                *["fs3", "fs4", constants.CEPHFILESYSTEM_SC],
+                marks=pytest.mark.polarion_id("OCS-XXX"),
+            ),
+        ],
```

Comment:
Please consider changing the project/pod names to be meaningful with 'project'/'pod' prefix

Reply:
The names need to be short in order to find them in HTML file

Reply:
![image](https://user-images.githubusercontent.com/61982127/133044625-1843f44c-5954-4402-96f2-ee0b7a5f0409.png)